source/manage/mmctl-command-line-tool.rst

```diff
@@ -76,6 +76,7 @@ mmctl usage notes
   - We recommend you add the path to the Mattermost ``bin`` folder into your ``$PATH`` environment variable. This ensures that you can run mmctl commands locally regardless of your current directory location.
   - If the ``bin`` directory is not added to the ``$PATH`` environment variable, each time you use mmctl you must be in the ``bin`` directory to run mmctl commands, and the commands must be prefixed with ``./``. If you're working from a different directory, make sure you specify the full path to mmctl when running mmctl commands.
 - Parameters in CLI commands are order-specific.
+- You can use the ``--local`` flag with mmctl commands to run them without authentication by allowing communicating with the server through a Unix socket.
```

Comment:
`--local` requires some config setup on the mattermost server to use, we'll probably want to include that https://docs.mattermost.com/manage/mmctl-command-line-tool.html#local-mode